packages/ag-charts-website/src/pages/[framework]/[pageName].astro

```diff
@@ -53,7 +53,14 @@ const descriptionWithSeoTagline = `${description} ${seoTagline}`;
     showSearchBar={true}
     showDocsNav={!hidePageMenu}
 >
-    <div class:list={[hidePageMenu && styles.noLeftMenu, styles.contentViewport, 'layout-grid']}>
+    <div
+        class:list={[
+            hidePageMenu && styles.noLeftMenu,
+            styles.contentViewport,
+            'layout-grid',
+            largeExamples && 'largeExamples',
```

Comment:
Another way to do conditional styles is:

```
{ largeExamples: largeExamples }

// Or just
{ largeExamples }
```

so,

```
<div
    class:list={[
        styles.contentViewport,
        'layout-grid',
        {
          [styles.noLeftMenu]: hidePageMenu,
          largeExamples
        }
    ]}
>
```

https://docs.astro.build/en/guides/styling/#combining-classes-with-classlist

Reply:
Good shout. Now updated